code/utils.py

```diff
@@ -53,11 +53,8 @@ def get_image_names(folder: Path) -> List[Path]:
     Returns:
         A list of the names with paths of the images in a folder.
     """
-    return sorted([
-        Path(f.name) for f in folder.iterdir() if ((
-            folder.joinpath(f.name).is_file()) and (".DS_Store" not in f.name))
-    ],
-                  key=str)
+    return sorted([Path(f.name) for f in folder.iterdir() if
+                   ((folder.joinpath(f.name).is_file()) and (".DS_Store" not in f.name) and (f.suffix.casefold() in IMAGE_EXTS))], key=str)
```

Comment:
checking ".DS_Store" is not necessary

Reply:
We still should check in case valid images are found within the ".DS_Store" folder.